ui/src/bibliographyGenerator/__tests__/BibliograpyGeneratorPageContainer.test.jsx

```diff
@@ -56,19 +76,23 @@ describe('BibliographyGeneratorPageContainer', () => {
         loading: false,
       }),
     });
+
+    render(
+      <Provider store={store}>
+        <BibliographyGeneratorPageContainer />
+      </Provider>
+    );
+
+    const { onSubmit } = BibliographyGenerator.mock.calls[0][0];
```

Comment:
Here it can be better if we can actually simulate the submission with fireEvent from rtl so as to simulate better the user behavior